Assignment2/NicolásOrtizSuaña.ttl

```diff
@@ -0,0 +1,18 @@
++@base <http://www.oeg-upm.net/ontologies/class/ >
++@prefix class: <http://www.ontologies.org/ontologies/class#>
++@prefix computer: <http://www.ontologies.org/ontologies/computer#>
++@prefix user: <http://www.ontologies.org/ontologies/user#>
++@prefix sensor: <http://www.ontologies.org/ontologies/sensor#>
++@prefix measurement: <http://www.ontologies.org/ontologies/measurement#>
++
++:Class01          class:includes :Sensor029;
++                  class:includes :Computer101.
++
++:Measurement8401  measurement:hasTemperature :"29"^^xsd:integer;
```

Comment:
It does not make sense to have ":" before the RDF literals. This is an error that also appeared in other files from other students, and I already recommended to remove those. Same in other lines